index.html

```diff
@@ -2,241 +2,277 @@
-          <li class="skill__section__container">
-            <div class="skill__info">
-              <h2 class="skill__title">Languages</h2>
-              <img src="Image/arrowdown.svg" alt="arrow-dropdowm">
-            </div>
-            <ul class="language__list">
-              <li class="language">
-                <img src="Image/JS.svg" alt="javascript">
-                <h2 class="language__name">JavaScript</h2>
-              </li>
-              <li class="language">
-                <img src="Image/HTML.svg" alt="Html">
-                <h2 class="language__name">HTML</h2>
-              </li>
-              <li class="language">
-                <img src="Image/CSS.svg" alt="Css">
-                <h2 class="language__name">CSS</h2>
-              </li>
-            </ul>
-          </li>
-          <li class="skill__info">
-            <h2 class="skill__title">Framework</h2>
-            <img src="Image/arrowright.svg" alt="arrow-dropdowm">
-          </li>
-          <li class="line"></li>
-          <li class="skill__info">
-            <h2 class="skill__title">Skills</h2>
-            <img src="Image/arrowright.svg" alt="arrow-dropdowm">
-          </li>
-          <li class="line"></li>
-        </ul>
-      </section>
-    
+        </li>
+        <li class="skill__info">
+          <h2 class="skill__title">Framework</h2>
+          <img src="Image/arrowright.svg" alt="arrow-dropdowm">
+        </li>
+        <li class="line"></li>
+        <li class="skill__info">
+          <h2 class="skill__title">Skills</h2>
+          <img src="Image/arrowright.svg" alt="arrow-dropdowm">
+        </li>
+        <li class="line"></li>
+      </ul>
+    </section>
+  </article>
+
+  <!-- Footer -->
+  <footer class="footer">
+    <h2 class="footer-title">Contact me</h2>
+    <p class="footer-text">If you have an application you are interested in developing, a feature that you need built or
+      a project that needs coding. I’d love to help with it</p>
+    <form class="form" action="https://formspree.io/f/xbjqvdnv" method="post">
+      <ul class="formul">
+        <li class="formul1">
+          <label for="fullName">
+            <input type="text" name="name" id="fullName" placeholder="Yeremias" maxlength="30" required>
+          </label>
+        </li>
+        <li class="formul2">
+          <label for="email">
+            <input type="email" name="email" id="email" placeholder="natajayanj14@gmail.com" required>
+          </label>
+        </li>
+        <li class="formul3">
+          <label for="message">
+            <textarea name="message" id="message" cols="30" rows="6" maxlength="500" placeholder="Write your message here"
+              required></textarea>
+          </label>
+        </li>
+        <li class="button-btn">
+          <button type="submit" class="btn">Get It Touch</button>
+        </li>
+      </ul>
+    </form>
+    <span class="line-bold"><img src="/Image/footer-line.svg" alt="line"></span>
```

Comment:
- [ ] You have done a great job with your contact form but there is an issue with the line that is below. It does not appear correctly. I think it is because you are using a `/` before the `SOURCE_PATH`. You could try deleting the `/` or even better using a `.` before the `/`, so it looks like this: `./`. This a recommended solution because this way when you want to go a directory up you could use two dots `../` :wink:

  ![image](https://user-images.githubusercontent.com/81522435/139755138-7e8814f8-41f8-4bf4-b058-db4777f4e3f4.png)